scala/private/rule_impls.bzl

```diff
@@ -166,8 +166,20 @@ def compile_java(ctx, source_jars, source_files, output, extra_javac_opts, provi
 def runfiles_root(ctx):
     return "${TEST_SRCDIR}/%s" % ctx.workspace_name
 
+def specified_java_runtime(ctx, default_runtime):
```

Comment:
```suggestion
def specified_java_runtime(ctx, default_runtime=None):
```

Reply:
so caller don't need to call with None

also would be good make sure the return value isn't None.